lib/pdk/validators/metadata/metadata_syntax.rb

```diff
@@ -42,7 +42,7 @@ def self.stop_spinner(exit_code)
       end
 
       def self.invoke(report, options = {})
-        targets = parse_targets(options)
+        targets, _skipped, _invalid = parse_targets(options)
```

Comment:
This class does not report on `_skipped` or `_invalid` and doesn't seem to call `super.invoke` - should reporting of skipped and invalid targets be added?